pipelines/rj_smas__disparo_pic/validators.py

```
@@ -0,0 +1,218 @@
+    Args:
+        destination_dict: Dicionário com dados do destinatário
+
+    Returns:
+        Tupla contendo:
+        - Bool indicando se é válido
+        - DestinationInput validado (ou None se inválido)
+        - String com mensagem de erro (vazia se válido)
+    """
+    try:
+        validated = DestinationInput(**destination_dict)
+        return True, validated, ""
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)